com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -15,12 +15,42 @@ namespace Unity.Netcode.Components
     [DefaultExecutionOrder(100000)] // this is needed to catch the update time after the transform was updated by user scripts
     public class NetworkTransform : NetworkBehaviour
     {
-        public const float PositionThresholdDefault = .001f;
-        public const float RotAngleThresholdDefault = .01f;
-        public const float ScaleThresholdDefault = .01f;
+        // Minimum threshold value for position, rotation, and scale
+        public const float ThresholdMinimum = 0.001f;
+        public const float PositionThresholdDefault = 0.001f;
+        public const float RotAngleThresholdDefault = 0.01f;
+        public const float ScaleThresholdDefault = 0.01f;
         public delegate (Vector3 pos, Quaternion rotOut, Vector3 scale) OnClientRequestChangeDelegate(Vector3 pos, Quaternion rot, Vector3 scale);
         public OnClientRequestChangeDelegate OnClientRequestChange;
 
+        protected const string k_PrecisionStringFormat = "0.0000000000";
+#if UNITY_EDITOR
+        // Make sure the rotation threshold is within the accepted range
+        // to "auto-correct"
+        private void OnValidate()
+        {
+            // Make sure all thresholds are greater than
+            // or equal to the ThresholdMinimum
+            if (PositionThreshold < ThresholdMinimum)
+            {
+                PositionThreshold = ThresholdMinimum;
```

Comment:
Well, this was my current train of thought on this:
We have a pre-release that allows users to set their threshold values to 0, and if users currently have any NetworkPrefabs (prefabs) with NetworkTransform threshold values already serialized as 0 then without OnValidate a user will have to manually edit the values for them to be kept within the minimum range.  So, for pre-existing users this will assure that in the event they have a threshold value set to zero it will be adjusted to the minimum threshold range automatically when they enter into playmode or make a build.

Reply:
+1 for warning logs to inform user here

Reply:
Added warning messages.